### PR TITLE
Fix bflix/Nites

### PR DESCRIPTION
**Pull request by swamproyal**
They moved to a new host, I guess we should still call the service bflix since thats what they use internally for domains and stuff?

### CODE REVIEW

File: lua/cinema/theater/services/sh_bflix.lua [hunk 1]
```diff
@@ -81,27 +83,34 @@ if CLIENT then
                     end
                 end)
 
+                -- The main video page
                 if url == key then
+                    self:QueueJavascript([[
+                        const title = document.querySelector('body div#aa-wp div.bd.cont section.section.single div.rw aside.left.cl1 article.post.single div.dfxb div header.entry-header h1.entry-title').textContent;
```
Comment (WinterPhoenix) [inline lua/cinema/theater/services/sh_bflix.lua]: this is way too specific. will almost certainly break

File: lua/cinema/theater/services/sh_bflix.lua [hunk 2]
```diff
@@ -119,28 +128,19 @@ if CLIENT then
     end
 
     function SERVICE:LoadVideo(Video, panel)
-        panel:EnsureURL(Video:Key())
+        panel:EnsureURL(Video:Data())
```
Comment (WinterPhoenix) [inline lua/cinema/theater/services/sh_bflix.lua]: You should be checking if Data ~= "" here. Data is just an empty string, not the Key, by default

**Review by WinterPhoenix — CHANGES_REQUESTED** [review]

File: lua/cinema/theater/services/sh_bflix.lua (hunk 2 above)
Comment (swamproyal) [reply lua/cinema/theater/services/sh_bflix.lua]: done

File: lua/cinema/theater/services/sh_bflix.lua (hunk 1 above)
Comment (swamproyal) [reply lua/cinema/theater/services/sh_bflix.lua]: I made it better/hopefully more robust

**Review by WinterPhoenix — APPROVED** [review]

**Pull request merged**